### DISCUSSION TITLE
Feature Support: Enable customization of the contentCollectionCache save location.

### DISCUSSION THREAD

**Discussion by justin5267** [withastro/roadmap#793] [Proposal]
When building static websites, users often directly package all files in the dist folder and upload them to hosting servers. However, currently, the cache files generated by the contentCollectionCache feature are also stored in the dist folder, requiring manual exclusion of relevant folders during the packaging and upload process.

**Comment by ematipico** [comment]
Hey @justin5267 , here's a suggestion: considering the fact that content collections cache RFC is still open, I think it's best to leave a **comment** in the PR of the RFC. That's the most appropriate way to leave feedback while a feature is still under the RFC process.

No need to open a new discussion unless a feature is already stable.

**Comment by florian-lefebvre** [comment]
The content collections cache flag has been removed in v5
